models/Element/Traits/VersionDaoTrait.php

```diff
@@ -60,10 +60,6 @@ public function getVersions()
             'ctype' => Element\Service::getElementType($this->model),
         ])->setOrderKey('id')->setOrder('ASC');
 
-        $versions = $list->load();
-
-        $this->model->setVersions($versions);
-
-        return $versions;
+        return $list->load();
```

Comment:
```suggestion
        return $list->getVersions();
```

Reply:
load method has been called before, so I wouldn´t necessarily change it. The property is set here for example:
https://github.com/pimcore/pimcore/blob/10.x/models/Asset.php#L1492 
What do you think? :)

Reply:
Right you are :)